testsuite/tests/singlecluster/authorino/identity/anonymous/test_anonymous_identity.py

```diff
@@ -31,8 +37,9 @@ def test_anonymous_identity(client, auth, authorization):
     response = client.get("/get")
     assert response.status_code == 401
 
+    generation = authorization.generation
     authorization.identity.add_anonymous("anonymous")
-    authorization.wait_for_ready()
+    authorization.wait_until(has_observed_generation(generation + 1))
```

Comment:
I would put the wait for `generation == observedGeneration` inside `wait_for_ready` in class `Policy` as it conceptually fits there.